libs/components/code-examples/src/lib/modules/forms/input-box/basic/example.component.spec.ts

```diff
@@ -12,20 +11,19 @@ import { FormsInputBoxBasicExampleComponent } from './example.component';
 
 describe('Basic input box example', () => {
   async function setupTest(options: { dataSkyId: string }): Promise<{
+    fixture: ComponentFixture<FormsInputBoxBasicExampleComponent>;
     inputBoxHarness: SkyInputBoxHarness;
     helpController: SkyHelpTestingController;
   }> {
     const fixture = TestBed.createComponent(FormsInputBoxBasicExampleComponent);
     const loader = TestbedHarnessEnvironment.loader(fixture);
     const helpController = TestBed.inject(SkyHelpTestingController);
+
     const inputBoxHarness = await loader.getHarness(
       SkyInputBoxHarness.with({ dataSkyId: options.dataSkyId }),
     );
 
-    fixture.detectChanges();
-    await fixture.whenStable();
-
-    return { inputBoxHarness, helpController };
+    return { fixture, inputBoxHarness, helpController };
```

Comment:
Don't call `fixture.detectChanges()` in the setup function because a test might want to validate the component's state before the first change detection cycle.